src/node/types/index.ts

```diff
@@ -14,3 +14,8 @@ export interface Footage {
   itemName: string;
   itemFolder: string;
 }
+
+export interface Settings {
+  apiKey?: string;
+  hasPin?: boolean;
+}
```

Comment:
I am not sure that it makes even sense that inside .ini file we make object strucutre;

because I think ini file is similar to env file for example
key=value
anotherKey=anotherValue

But I did it this way because of `JSON.parse`, or do we even care, and make it look the way we want?